cmd/skupper/skupper.go

```diff
@@ -440,10 +441,26 @@ installation that can then be connected to other skupper installations`,
 				return fmt.Errorf("The --enable-flow-collector option must be used with the --enable-console option")
 			}
 
+			if routerCreateOpts.AuthMode != "internal" && (len(routerCreateOpts.User) > 0 || len(routerCreateOpts.Password) > 0) {
+				return fmt.Errorf("the --console-auth option must be set to internal")
+			}
+
+			if routerCreateOpts.AuthMode == "internal" && (len(routerCreateOpts.User) == 0 || len(routerCreateOpts.Password) == 0) {
+				return fmt.Errorf("the user and password values need to be configured if the --console-auth is set to internal")
```

Comment:
This isn't the case. Specifying the password in the options is an insecure convenience only. The username(s) and password(s) can be directly specified through the skupper-console-users secret also (probably the recommended option), or just the username can be specified in which case the password will be randomly generated.

Reply:
Skupper behaviour is correct, if the user is set but not the password, it generates a random password that it is not an empty value. Updated in b2bd80797a311e98cad54e4793baed3b2244ef91